apps/dotcom/styles/z-board.css

```diff
@@ -317,3 +317,70 @@
 .tlui-layout[data-breakpoint='0'] .tlui-offline-indicator {
 	margin-top: 4px;
 }
+
+/* Document Name */
+
+.tlui-document-name__inner {
+	border-radius: calc(var(--radius-2) + var(--space-2));
+	background-color: var(--color-background);
+	padding: 0px 0px 0px 0px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	height: 40px;
+	max-width: 100%;
+	color: var(--color-text);
+	text-shadow: 1px 1px 0px var(--color-background);
+	pointer-events: all;
+}
+
+.tlui-document-name__input__wrapper {
+	position: relative;
+	max-width: calc(100% - 40px);
+	display: flex;
+	flex: auto;
+}
+
+.tlui-document-name__input,
+.tlui-document-name__text {
+	padding: var(--space-2) var(--space-3);
+	white-space: pre;
+	line-height: 24px;
+}
+
+.tlui-document-name__input {
+	position: absolute;
```

Comment:
styling thing: feels like the input field should have a min-width of some sort. when typing in the first time it starts out as 60px but then goes to the single width of the character which is slightly jarring